### ISSUE TITLE
Default Checks Included when using -configuration flag in get/invoke-sqlassessment

### ISSUE THREAD

**Issue by jlent01** [microsoft/sql-server-samples#633]
When running the following command I get checks that are not included in 'databaseConfig.json' 
$databases = Get-SqlDatabase -ServerInstance 'localhost'
$checks = Get-SqlAssessmentItem $databases -Configuration "C:\SQL-Assessment-API\databaseConfig.json" | Select Id, Description -Unique 
$checks

The same is true when I use the invoke-sqlassessment cmdlet.

Is this expected behavior?  How would I pass only the checks associated with my config file?  The only way I've found so far is to pupulate my $checks in conjunction with out-gridview -passthough, to filter out what I don't want.

Output from my command:
![defaults with custom](https://user-images.githubusercontent.com/28069508/64369587-8b147900-cfd1-11e9-8c3b-356e3bcdc7e2.gif)

Content of my config.json:
![myConfig json](https://user-images.githubusercontent.com/28069508/64369730-d2026e80-cfd1-11e9-83ac-20af71ab3235.gif)

**Comment by alexprotsenko** [comment]
Yes, it's expected. The cmdlets run checks of the default ruleset by default. You need to disable them all in your json (you can do it in either the json with your checks or a separate json).  

I got it for you but found that one check - DeprecatedFeatures - doesn't get disabled for some reason. It's a bug, we'll fix it in the next version. If your checks target at Database only, this issue won't affect you. Also, in the next version, all the default checks will have a special tag so you can disable them easily. 

`{ 
	"checks":[ 
		{ 
			"select": ["TraceFlag"], 
			"enabled": false 
		},
		{ 
			"select": ["Performance"], 
			"enabled": false 
		},
		{ 
			"select": ["Update"], 
			"enabled": false 
		},
		{ 
			"select": ["deprecated_feature"], 
			"enabled": false 
		},
		{ 
			"select": ["XEvent"], 
			"enabled": false 
		},
		{ 
			"select": ["DefaultCollections"], 
			"enabled": false 
		},
		{ 
			"select": ["Hints"], 
			"enabled": false 
		},
		{ 
			"select": ["Plan"], 
			"enabled": false 
		},
		{ 
			"select": ["Hypothetical"], 
			"enabled": false 
		}	
	] 
}`

@aguzev for awareness

**Comment by aguzev** [comment]
There is another way to run only your checks. Just put a tag on every check like:
`  'tags': ["MyChecks']`
Then you can get only your checks by 
`Get-SqlAssessmentItem -Check 'MyChecks'
Invoke-SqlAssessment -Check 'MyChecks'`

**Comment by alexprotsenko** [comment]
@aguzev good one

**Comment by jlent01** [comment]
Great, thanks Aleksei and Alex.  I'll give it a try.
